src/backend/utils/misc/guc_gp.c

```diff
@@ -3408,6 +3410,17 @@ struct config_bool ConfigureNamesBool_gp[] =
 		NULL, NULL, NULL
 	},
 
+	{
+		{"gp_add_partition_inherits_table_setting", PGC_USERSET, CUSTOM_OPTIONS,
+			gettext_noop("Alter table add partition inherits storage setting from the table."),
```

Comment:
I suggest to write the query in capital letters like here https://github.com/arenadata/gpdb/blob/922dec714acfe11fd5ff3735a669c84e7a69ed6d/src/backend/utils/misc/guc_gp.c#L672 and clarify that the GUC affects AO tables only
```suggestion
			gettext_noop("ALTER TABLE ADD PARTITION inherits storage setting from the AO table."),
```

Reply:
applied